### ISSUE TITLE
Can't empty Nuclearcraft input tanks

### ISSUE THREAD

**Issue by ZariusCorten** [DevDyna/IronBerry#21]
### Core Section

- [X] I have searched for this issue on this repository
- [X] This bug persist and I need help

### Environment

Singleplayer/Local | NO LAN

### Modpack Version

0.9b

### Description

NuclearCraft machines do not allow emptying/extraction of fluids in the input tanks. This is a problem if I want to use the a machine for more than one thing. NuclearCraft used to have the option to use a metakey+right click to empty tanks, so problem may be with the mod rather than the implementation.

### Procedure

Add fluid/gas to an input tank. Try to remove it.

### Additional things

N/A

**Comment by DevDyna** [comment]
> NuclearCraft machines do not allow emptying/extraction of fluids in the input tanks

Yeah.. it isn't an unknown issue due NC:N still returning too many issues probably more that I was expected...
